contributions/presentation/week3/vemula-akhily/Readme.md

```diff
@@ -0,0 +1,28 @@
+# Presentation Proposal :  "An Introduction to Application-level Chaos Engineering and Its Tools".
+
+## Members
+Nagasudeep Vemula, vemula@kth.se
+Github:nagavem
+
+Akhil Yerrapragada, akhily@kth.se
+Github: akhilyerrapragadaa
+
+## Description:
+
+ -Introductions to fault engineering and chaos engineering and a comparison between them.
+ -Performace evaluation of a spring application using chaos machine/chaos monkey.
+ -A look into the tools that would be used for chaos engineering.We will delve into the different use cases for them and the different categories of tools that are available at the moment.
+ 
+ -Relevance to Devops: With regards to the relevance, Adding continuous chaos to DevOps culture helps us build better anti-fragile applications.
+ -Technical and business metrics to be considered.
+ -Chaos Experiments to be run.
+ -And finally, Chaos engineering maturity.
+ 
+ 
+ ## References:
+ 
+ [Chaos Machine Deep Dive](https://doi.org/10.1109/TSE.2019.2954871)
+ [Chaos Engineering in Devops] (https://medium.com/capital-one-tech/continuous-chaos-introducing-chaos-engineering-into-devops-practices-75757e1cca6d)
```

Comment:
incorrect Markdown links here (space between square bracket and parenthesis), could you fix please? you may add bullets also. Thanks!